bastion-gui.py

```
@@ -481,36 +481,54 @@ def show_update_notification(self):
+            deb_path = None
+            try:
+                fd, deb_path = tempfile.mkstemp(prefix="bastion-update-", suffix=".deb")
+                os.close(fd)
+                urllib.request.urlretrieve(url, deb_path)
+                os.chmod(deb_path, 0o644)
+                print(f"[UPDATE] Installing v{version}...")
+                # argv list, never a shell string: no version data is
+                # interpreted by a shell.
+                proc = subprocess.Popen(['pkexec', 'apt-get', 'install', '-y', deb_path])
+                proc.wait()
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)